WORKSPACE.yaml

```diff
@@ -22,6 +22,7 @@ defaultArgs:
   jbBackendVersion: "latest"
   REPLICATED_API_TOKEN: ""
   REPLICATED_APP: ""
+  dockerVersion: 20.10.17
```

Comment:
@kylos101 @aledbf The only change for the workspace team here is to extract docker version from docker-up to here. That we can bundle docker CLI of the same version in supervisor as a fallback if a user does not have one.

We would liek to land this PR after new cluster deployment. Are you fine with this change?

Reply:
@akosyakov can you add `.workspace.yaml` to the codeowners file and assign each product team?

:+1: for the Single Source of Truth

Reply:
@kylos101 would it be alright to create another PR for this file to consider ownership change.

We would like to land it I would like to avoid off-topic discussion on this PR 🙏 

Also if you are fine with moving docker version there could you give an approve from the workspace team please 🙏

Reply:
> Also if you are fine with moving docker version there could you give an approve from the workspace team please pray

@akosyakov this PR is still in a draft state, is it ready for review?

> @kylos101 would it be alright to create another PR for this file to consider ownership change.

Yes :+1: